src/CommandingOfficers/Tech.java

```diff
@@ -150,10 +143,12 @@ public OverchargeAbility(Commander commander, String abilityName, double ability
       defenseBuff = new CODefenseModifier(buff);
       healAmount = healAmt;
 
-      // Only mechanical units get the firepower boost.
+      // Only mechanical/non-troop units get the firepower boost.
+      long allUnits = ~0;
       long exclude = UnitModel.TROOP;
-      ArrayList<UnitModel> models = commander.getAllModels(Tech.mechanicalUnits, true, exclude);
-      for( UnitModel m : models )
+      unitsToOverCharge = commander.getAllModels(allUnits, true, exclude);
```

Comment:
Maybe introduce a new `getAllModelsNot(long exclude)`, since that would read more clearly IMO.